Podcast/DiscoverComponentViewController.swift

```diff
@@ -62,4 +67,13 @@ class DiscoverComponentViewController: ViewController, NVActivityIndicatorViewab
         return header
     }
 
+    @objc func pullToRefresh() {
```

Comment:
Why can't we just override this method?

Reply:
i had that before and it was throwing an error before but now it's not... weird